ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -498,6 +564,17 @@ private static DescribeTopicsResult describeTopicReturningUnknownPartitionExcept
     return describeTopicsResult;
   }
 
+  private static DescribeTopicsResult describeTopicTopicAuthorizationException(String topicName) {
```

Comment:
super nit:
```suggestion
  private static DescribeTopicsResult describeTopicAuthorizationException(String topicName) {
```
to be consistent with `createTopicAuthorizationException()` below.